test/rubygems/test_gem_spec_fetcher.rb

```diff
@@ -168,20 +168,22 @@ def src.fetch_spec(name)
   def test_suggest_gems_from_name_latest
     spec_fetcher do|fetcher|
       fetcher.spec "example", 1
-      fetcher.spec "other-example", 1
+      fetcher.spec "an-example", 1
       fetcher.spec "examp", 1
```

Comment:
Given the short gem name situation, we should add a lookup for `"amp"` that returns nothing, and `"exam"` that returns `"examp"`.

Reply:
Done.